scripts/apps/authoring-react/authoring-angular-integration.tsx

```diff
@@ -144,11 +144,46 @@ function getInlineToolbarActions(options: IExposedFromAuthoring<IArticle>): IAut
         availableOffline: true,
     });
 
+    const getReadOnlyAndArchivedFrom = (): Array<ITopBarWidget<IArticle>> => {
```

Comment:
what about states at lines 327-370? It should have read-only archived indicators too. It would probably be good to have some `commonActions` that would be everywhere.

Reply:
I plant to address those lines in the next bulltet point of the ticket which is about all of these other cases.